src/main/java/qna/domain/BaseTimeEntity.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public abstract class BaseTimeEntity {
```

Comment:
```suggestion
public class BaseTimeEntity {
```

이 클래스를 상속용으로만 사용하라는 의미로 abstract 를 사용하신 것 같아요.
이미 붙은 어노테이션이 있어서 굳이 붙이지 않아도 괜찮지 않을까요?